### PR TITLE
fix hyperlink pointing

### PR DESCRIPTION
**Pull request by nivance**
## What is the purpose of the change

fix hyperlink pointing in what-is-nacos.html

## Brief changelog

issue is [here](https://github.com/nacos-group/nacos-group.github.io/issues/118)

### CODE REVIEW

File: docs/en-us/what-is-nacos.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Key features of Nacos:
 
     Centralized management of configuration makes it more convenient for you to achieve stateless services and elastic expansion of service instances on-demand.
 
-    Nacos provides an [easy-to-use UI TODO](xx) to help you manage all of your application or services's configurations. It provides some out-of-box features including configuration version tracking, canary/beta release, configuration rollback, and client configuration update status tracking to ensure the safety and control the risk of configuration change. 
+    Nacos provides an [easy-to-use UI](http://console.nacos.io/nacos/index.html) to help you manage all of your application or services's configurations. It provides some out-of-box features including configuration version tracking, canary/beta release, configuration rollback, and client configuration update status tracking to ensure the safety and control the risk of configuration change.
```
Comment (KomachiSion) [inline docs/en-us/what-is-nacos.md:43]: Should we add an DEMO word?

like `Nacos provides an easy-to-use UI ([DEMO](http://console.nacos.io/nacos/index.html))`

Comment (nivance) [reply docs/en-us/what-is-nacos.md:43]: which one is better?

Comment (KomachiSion) [reply docs/en-us/what-is-nacos.md:43]: I'm afraid that the second one will mislead readers into thinking that this demo is the UI itself.

Just for showing way, the second  one is more concise

Comment (KeRan213539) [reply docs/en-us/what-is-nacos.md:43]: > Should we add an DEMO word?
> 
> like `Nacos provides an easy-to-use UI ([DEMO](http://console.nacos.io/nacos/index.html))`

I agree with this

Comment (nivance) [reply docs/en-us/what-is-nacos.md]: ok, I have modified it

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**